modules/ROOT/pages/dialog-configuration.adoc

```diff
@@ -62,3 +67,47 @@ For more information on the `+dialogApi+` object that is passed to some of the c
 Each of the event callback functions - `+onAction+`, `+onSubmit+`, `+onCancel+`, `+onChange+`, `+onClose+`, and `+onTabChange+` - are shared between all dialog components that may trigger them. For example, *Custom* type footer buttons and dialog panel buttons all trigger `+onAction+`. Therefore, callback functions that may be triggered by multiple components are passed an object (called `+details+` above) that contains the `+name+` of the component that triggered the event.
 
 Any callback function that is not passed a `+details+` object assumes that the dialog will only contain one component which can trigger it or that it does not matter if the function is triggered by multiple components. For example, `+onSubmit+` is only triggered when a user clicks on a *Submit* type footer button, and {productname} assumes that a dialog will only have one *Submit* type button. In comparison, `+onCancel+` and `+onClose+` are both triggered by clicking the `+X+` button in the top right of a dialog or by clicking a *Cancel* type footer button. These two buttons have the same functionality, and therefore {productname} does not differentiate between them.
+
+== Dialog position
+
+By default a dialog presents as a modal object in the center of the editor viewport.
+
+This presented position can be changed for a given dialog by adding an `inline` value to the dialog’s configuration.
```

Comment:
Actually, `{inline: 'bottom'}` should be provided in the second argument to windowManager.open() as seen in the example below

Reply:
@tiny-ben-tran,

> Actually, `{inline: 'bottom'}` should be provided in the second argument to windowManager.open() as seen in the example below

¿que?

If, by _example below_, you mean the example I added, said example doesn’t have an `{inline: 'bottom'}` value in it at all.

What am I missing?

Reply:
Sorry. I meant `inline: 'toolbar'`.

Reply:
Appropriate edits done.